keynav.c

```diff
@@ -216,7 +217,7 @@ dispatch_t dispatch[] = {
 
   // Mouse activity
   "warp", cmd_warp,
-  "click", cmd_click,     
+  "click", cmd_click,
```

Comment:
The trailing whitespace changes are making the diff harder to read for me, so I am likely to miss some things in review.

Reply:
Sorry about that, my emacs config automatically trims it.

Reply:
Maybe put all traling whitespace changes into a single commit at the very end of the list of patches.